### PR TITLE
Update site

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,67 @@
...
+.io/siddhi">Siddhi</a> that provides the functionality to convert canonical events of the server in the WSO2Event format to any text message format and convert events of any text format to the server's canonical event format (WSO2Event) for processing.
+
+Find some useful links below:
+
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-text">Source code</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-text/releases">Releases</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-text/issues">Issue tracker</a>
+
+## Latest API Docs 
+
```
Comment (tishan89) [inline README.md]: Latest API docs need to come here. Refer unique docs

File: mkdocs.yml [hunk 2]
```diff
@@ -9,14 +9,12 @@ extra:
...
     primary: teal
     accent: teal
+google_analytics:
+- UA-103065-28
+- auto
 pages:
-- Welcome to WSO2 Siddhi IO TCP: index.md
-- Api:
-  - 2.0.0-SNAPSHOT: api/1.0.0-SNAPSHOT.md
+- Welcome to WSO2 Siddhi Execution Unique: index.md
```
Comment (tishan89) [inline mkdocs.yml]: unique??

File: README.md (hunk 1 above)
Comment (Yasara123) [reply README.md]: Fix in d6dc7df

File: mkdocs.yml (hunk 2 above)
Comment (Yasara123) [reply mkdocs.yml]: Fixed in d6dc7df

**Pull request merged**